windows/CodePushInvalidUpdateException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Since we're not actually augmenting the error object in any way, I still vote for us using the InvalidOperationException class (or another one as appropriate). I believe that C# favors re-using core exception types as much as possible, and I think we can achieve the behavior we want of rejecting a bad download without needing to subclass our our exception type.